src/content/docs/accounts/accounts/account-maintenance/promotions-ui.mdx

```diff
@@ -0,0 +1,104 @@
+---
+title: Product promotions 
+tags:
+  - Accounts
+  - Accounts and billing
+  - General account settings
+metaDescription: 'How to sign up for product promotions'
+freshnessValidatedDate: 2024-07-15 
+---
+
+
+import accountsPreviewsandTrials from 'images/accounts_screenshot-crop_previews-and-trials.webp'
+
+import accountsPromotionsTable from 'images/accounts_screenshot-crop_promotions-table.webp'
+
+import accountsManagePreview from 'images/accounts_screenshot-crop_manage-preview.webp'
+
+import accountsPromoUpgradeButton from 'images/accounts_screenshot-crop_promo-upgrade-button.webp'
+
+
+
+Promotions are special offers to try out New Relic product features. You can browse through our various promotions and manage your enrollment in the UI at **Previews & trials**. 
+
+The type of promotion currently available is the product preview. This is the opportunity to try out a feature before it is released. Here's what what you get in the UI:
+
+* One-stop shopping for previews: View all available product previews in one place.
+* Enrollment flexibility: Easily sign-up or leave from product previews until they become generally available (GA).
+* Upgrade options: Upgrade to access products once they're GA.
+
+To help you get started, we've provided some tips about eligibility and enrollment.
+
+## Eligibility [#eligibility]
+
+You can check the plan management page to see if your organization is eligible for a product promotion. If you don't see the promotions UI, your organization isn't eligible.
```

Comment:
```suggestion
You can check the plan management page to see if your organization is eligible for a product promotion. If you don't see the Previews & trials UI, your organization isn't eligible.
```

Reply:
@ally-sassman I a subsequent commit, I added the path in the first sentence of the paragraph, and then in the second sentence, I switched it to "If you don't see this page, your organization isn't eligible."